extensions/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+## 5.0.1 - 2023-04-20
+
```

Comment:
Feels a bit weird because this is completely empty...

But according to the [guiding principles][1]:

> There should be an entry for every single version.

[1]: https://keepachangelog.com/en/1.1.0/#how